RecoLocalTracker/Phase2TrackerRecHits/plugins/Phase2TrackerRecHits.cc

```diff
@@ -63,8 +63,8 @@ void Phase2TrackerRecHits::produce(edm::StreamID sid, edm::Event& event, const e
   eventSetup.get< TrackerDigiGeometryRecord >().get(geomHandle);
   const TrackerGeometry* tkGeom(&(*geomHandle));
 
-  edm::ESHandle< TrackerTopology > tTopoHandle;
-  eventSetup.get< IdealGeometryRecord >().get(tTopoHandle);
+  //edm::ESHandle< TrackerTopology > tTopoHandle;
+  //eventSetup.get< IdealGeometryRecord >().get(tTopoHandle);
```

Comment:
just commit in your branch  you used to make the pR (lowette:Phase2TrackerLocalReco) the changes and push , the PR will be updated